ui/next.config.js

```diff
@@ -1,4 +1,18 @@
 /** @type {import('next').NextConfig} */
+const CSPDirectives = [
+    "default-src 'self';",
+    "script-src 'self' 'unsafe-eval' 'unsafe-inline';",
+    "style-src 'self' 'unsafe-inline';",
+    `connect-src 'self' *.api.octopus.ac http://127.0.0.1:4003 https://api.octopus.ac https://api.ror.org https://cdn.contentful.com;`,
+    'img-src https:;',
```

Comment:
Should this have a defined list?

Reply:
I would suggest not because there's only this place that it's being used, but if we needed it elsewhere a dedicated list would make more sense